"jackey.h missing" build error

When building on a fresh machine with:

```
mkdir build
cd build
cmake -DSC_DOC_RENDER=False -DSUPERNOVA=True -DNATIVE=True -DSC_IDE=False -DSC_QT=False -DSCLANG_SERVER=False -DSC_WII=False -DSC_ED=False -DSC_VIM=False -DNO_AVAHI=True -DAUDIOAPI=jack ..
make
```


> Scanning dependencies of target server_test
> [ 96%] Building CXX object testsuite/supernova/CMakeFiles/server_test.dir/server_test.cpp.o
> In file included from /home/air/box_build/supercollider/server/supernova/./server/server.hpp:40:0,
>                  from /home/air/box_build/supercollider/testsuite/supernova/server_test.cpp:3:
> /home/air/box_build/supercollider/server/supernova/./audio_backend/jack_backend.hpp:31:24: fatal error: jackey.h: No such file or directory
>  #    include <jackey.h>
>                         ^
> compilation terminated.
> testsuite/supernova/CMakeFiles/server_test.dir/build.make:62: recipe for target 'testsuite/supernova/CMakeFiles/server_test.dir/server_test.cpp.o' failed
> make[2]: *** [testsuite/supernova/CMakeFiles/server_test.dir/server_test.cpp.o] Error 1
> CMakeFiles/Makefile2:4651: recipe for target 'testsuite/supernova/CMakeFiles/server_test.dir/all' failed
> make[1]: *** [testsuite/supernova/CMakeFiles/server_test.dir/all] Error 2
> Makefile:160: recipe for target 'all' failed
> make: *** [all] Error 2
> 


(Note that jackey.h does in fact exist, at external_libraries/jackey/jackey.h)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -143,6 +143,16 @@ endif()
 
 set(AUDIOAPI "default" CACHE STRING "Audio API to use (one of {default,coreaudio,jack,portaudio})")
 
+if (AUDIOAPI STREQUAL jack)
+	# here we check for JACK metadata API
+	include(CheckIncludeFiles)
+	CHECK_INCLUDE_FILES("jack/metadata.h" JACK_USE_METADATA_API)
+	if(${JACK_USE_METADATA_API})
+		message(STATUS "using JACK metadata API")
+	    include_directories(${CMAKE_SOURCE_DIR}/external_libraries/jackey)
+	endif()
+endif()
+
 option(LIBSCSYNTH "Compile libscsynth as shared library" OFF)
 
 option(INSTALL_HELP "Install help docs and examples along with the software" ON)

--- a/server/CMakeLists.txt
+++ b/server/CMakeLists.txt
@@ -34,16 +34,6 @@ else()
 	endif()
 endif()
 
-if (AUDIOAPI STREQUAL jack)
-	# here we check for JACK metadata API
-	include(CheckIncludeFiles)
-	CHECK_INCLUDE_FILES("jack/metadata.h" JACK_USE_METADATA_API)
-	if(${JACK_USE_METADATA_API})
-		message(STATUS "using JACK metadata API")
-		include_directories(${PROJECT_SOURCE_DIR}/external_libraries/jackey)
-	endif()
-endif()
-
 add_subdirectory(plugins)
 add_subdirectory(scsynth)
 

